steps/install-bundler/1.2.0/step.yml

```diff
@@ -0,0 +1,39 @@
+title: Install specific bundler version
+summary: This step allows you to uninstall the system bundler version, and match the
+  one in your Gemfile.
+description: This step allows you to uninstall the system bundler version, and match
+  the one in your Gemfile..
+website: https://github.com/FutureWorkshops/bitrise-step-install-bundler
+source_code_url: https://github.com/FutureWorkshops/bitrise-step-install-bundler
```

Comment:
What is the reason for removing `support_url`? Our automatic checks do not make it required, however, if not provided and the step is failing the step summary footer box will show `Not provided` for the Support URL.